kratos/python/add_serializer_to_python.cpp

```diff
@@ -25,8 +25,47 @@
 
 namespace Kratos
 {
+    
 namespace Python
 {
+
+    //This is a Helper object to provide a simpler interface for serialization to a string instead of to a file
+    class StreamSerializer : public Serializer 
```

Comment:
I would suggest to add this class in Internals namespace in order to show that it is not for external use....